src/index.js

```diff
@@ -122,20 +143,66 @@ class EdfContentScript extends ContentScript {
 
   async fetch(context) {
     this.log('info', 'fetch start')
+    const { sourceAccountIdentifier, manifest, trigger } = context
+
+    // force fetch all data (the long way) when last trigger execution is older than 30 days
+    // or when the last job was an error
+    const isLastJobError =
```

Comment:
In the future, can we think of doing all those things directly in the library ?
That way we can just call a function returning an object with `isLastJobError`, `distanceInDays` and `FORCE_FETCH_ALL` (for this last one I assume it's not mandatory regarding the fact there's a "TODO" comment mentionning this should be a flag). 
Just avoiding to do this *inside* the konnector, as we'll need this for all of them ?

Reply:
Yes, this is planned. We will put as much a possible what is generic in the library. But this is a first step

Reply:
Great, will be simpler

We don't even need distanceInDays in the konnector in this cas. But maybe it should be configurable in manifest.

Reply:
Yes the plan would be to be able to force this FETCH_ALL from even from bender or even by the user